py/private/py_image_layer.bzl

```diff
@@ -0,0 +1,156 @@
+"""py_image_layer macro for creating multiple layers from a py_binary
+
+>> [!WARNING]
```

Comment:
double `>` intentional?

Reply:
nope, i was trying to get stardoc leave it alone, it seem to escaping it, aware of any workarounds?

Reply:
hm that is very interesting, I haven't had that problem before. WHatever it's just cosmetic and inconsequential, ship it :)